xwalk_core_library_android.gypi

```diff
@@ -466,8 +472,11 @@
           ],
           'action': [
             'python', '<(DEPTH)/xwalk/build/android/generate_xwalk_core_library_aar.py',
-            '-t', '<(PRODUCT_DIR)',
-            '--shared',
+            '--aar-path', '<(PRODUCT_DIR)/xwalk_shared_library.aar',
+            '--android-manifest', '<(DEPTH)/xwalk/build/android/xwalk_shared_library_template/AndroidManifest.xml',
+            '--classes-jar', '<(PRODUCT_DIR)/lib.java/xwalk_core_library_java_app_part.jar',
+            '--res-dir', '<(PRODUCT_DIR)/xwalk_shared_library/res',
+            '--r-txt', '<(PRODUCT_DIR)/xwalk_core_internal_empty_embedder_apk/gen/R.txt',
```

Comment:
I know it has nothing to do with this PR.
Why can't we just remove the 'output' if no stamp is needed?  Is the 'output' is mandatory in "action" of GYP?